cmd/cloud_sql_proxy/proxy.go

```diff
@@ -292,7 +292,7 @@ func parseInstanceConfigs(dir string, instances []string, cl *http.Client) ([]in
 // CreateInstanceConfigs verifies that the parameters passed to it are valid
 // for the proxy for the platform and system and then returns a slice of valid
 // instanceConfig.
-func CreateInstanceConfigs(dir string, useFuse bool, instances []string, instancesSrc string, cl *http.Client) ([]instanceConfig, error) {
+func CreateInstanceConfigs(dir string, useFuse bool, instances []string, instancesSrc string, maxConnections uint64, cl *http.Client) ([]instanceConfig, error) {
```

Comment:
Actually, just noticed: it doesn't appear that you need to modify this function anymore. Can you revert this change to this file? (unless I'm misreading)